src/virtualenv/config/ini.py

```diff
@@ -0,0 +1,68 @@
+from __future__ import absolute_import, unicode_literals
+
+import logging
+import os
+
+from virtualenv.info import PY3, get_default_config_dir
+from virtualenv.util import ConfigParser
+from virtualenv.util.path import Path
+
+from .convert import convert
+
+DEFAULT_CONFIG_FILE = get_default_config_dir() / "virtualenv.ini"
```

Comment:
Is it too late to bikeshed what config format the file is in :)?

Reply:
you can make your case, but wanted something that has stdlib support...

Reply:
That indeed makes things very limited yeah :)

I think all the cases have been made at this point so I'll refrain unless someone else wants to continue painting the shed :D